data_steward/analytics/cdr_ops/retraction_in_place_validation_sandbox.py

```diff
@@ -0,0 +1,305 @@
+    table_check_query = JINJA_ENV.from_string('''
+  SELECT
+    \'{{table_name}}\' AS table_name,
+    {% if domain_id != 'ehr' and domain_id != 'person' and domain_id != 'death' and domain_id != ''%}
+        Case when count(tb.{{domain_id}}_id) = 0 then 'OK'
+        ELSE
+        'PROBLEM' end as retraction_status,
+        'EHR_domain_id' as source,
+        '{{domain_id}}' as domain
+    {% elif domain_id == 'death' %}
+        Case when count(tb.person_id) = 0 then 'OK'
+        ELSE
+        'PROBLEM' end as retraction_status,
+        'person_id' as source,
+        '{{domain_id}}' as domain
+    {% elif domain_id == 'person' %}
+        Case when count(tb.person_id) = 0 then 'OK'
+        ELSE
+        'OK' end as retraction_status,
+        'person_id' as source,
+        '{{domain_id}}' as domain
+    {% elif domain_id == 'ehr' or domain_id == '' %}
+        Case when count(tb.person_id) = 0 then 'OK'
+        ELSE
+        'OK' end as retraction_status,
+        'person_id' as source,
+        'UNKNOWN' as domain
+    {% endif %}
+  FROM
+    `{{project}}.{{dataset}}.{{table_name}}` as tb
+    right JOIN
+        pids as p
+    USING(person_id)
+    {% if domain_id != 'ehr' and domain_id != 'person' and domain_id != 'death' and domain_id != ''%}
+      where {{domain_id}}_id > 2000000000000000
+    {% endif %}
+  ''')
+
+    queries_list = []
+    is_deidentified = str('deid' in dataset).lower()
+
+    for table in pid_table_list:
+        domain_id = ''
+        if 'ehr' in table:
+            domain_id = 'ehr'
+        elif 'person' in table:
+            domain_id = 'person'
+        elif 'death' in table:
+            domain_id = 'death'
+        else:
+            for name in domain_tables:
+                if name in table:
+                    if 'observation' in name:
+                        if 'observation_period' in table:
+                            domain_id = 'observation_period'
+                        else:
+                            domain_id = 'observation'
+                        break
+                    else:
+                        domain_id = name
+                        break
+
+        queries_list.append(
+            table_check_query.render(project=project_id,
+                                     dataset=dataset,
+                                     table_name=table,
+                                     domain_id=domain_id))
+
+    union_all_query = '\nUNION ALL\n'.join(queries_list)
+
+    retraction_status_query = (f'{rids_query}\n{union_all_query}'
+                               if is_deidentified == 'true' else
+                               f'{pids_query}\n{union_all_query}')
+    result = execute(client, retraction_status_query)
+    all_results.append(result)
+
+for result, dataset in zip(all_results, datasets):
+    print(dataset)
+    ICD.display(result)
+    print("\n")
+# -
+
+# ## 2. Verify Row counts of source dataset minus the retracted participants  data is equal to the
```

Comment:
It looks like this sentence is cut in the middle. It should be something like this:

`2. Verify that the difference between before and after the retraction is only the participants listed to be dropped in the lookup table.`